src/common/txsplain.js

```diff
@@ -597,6 +597,36 @@ angular.module('txsplain', [])
             ' offer #<b>' + tx.tx.OfferSequence + '</b>.'
         }
 
+        // renderEscrowCreate
+        function renderEscrowCreate() {
+          var html = ''
+          if (tx.tx.Destination && tx.tx.Destination !== tx.tx.Account) {
+            html = 'The escrow is from <account>' + tx.tx.Account +
+              '</account> to <account>' + tx.tx.Destination + '</account><br/>'
+          }
+          html += 'It escrowed ' +
+            '<amount>' + displayAmount(tx.tx.Amount) + '</amount>'
+          if (tx.tx.Condition)
+            html += '<br/>Condition: ' + tx.tx.Condition
+          if (tx.tx.CancelAfter) {
+            html += '<br/>It can be cancelled after <date>' +
+              moment((RIPPLE_EPOCH + tx.tx.CancelAfter)*1000).format('LLL') +
+              '</date>.'
+          }
+          if (tx.tx.FinishAfter) {
+            html += '<br/>It can be finished after <date>' +
+              moment((RIPPLE_EPOCH + tx.tx.FinishAfter)*1000).format('LLL') +
+              '</date>.'
+          }
+          return html
+        }
+
+        // renderEscrowFinish
+        function renderEscrowFinish() {
```

Comment:
looks like all the necessary info is in the deleted escrow node in the metadata, I am going to write up a more detailed description to output that I think will clarify some potentially confusing things.